### ISSUE TITLE
wow

### ISSUE THREAD

**Issue by olsonbj** [bartzc/csPortfolio#2]
wow very cool chris thank you

**Comment by BensonOmb** [comment]
I agree with this and I think there is a lot of potential. The portfolio itself was nicely done, but some of the projects arn't working. They had good ideas just didn't work. 15/15
